### PR TITLE
adds recently viewed with hardcoded sample data

### PR DESCRIPTION
**Pull request by vtMMS**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:

- Before: https://main--mmsg-oly--aemsites.hlx.live/
- After: https://car-card-recentlyviewed--mmsg-oly--aemsites.hlx.live/

### CODE REVIEW

**Comment by vtMMS** [comment]
Hi @mrugesh-adobe , please go ahead and merge this one if you're happy with everything.

File: blocks/wishlist/wishlist.css [hunk 1]
```diff
@@ -39,6 +39,16 @@
   gap: 1rem;
 }
```
Comment (mrugesh-adobe) [inline blocks/wishlist/wishlist.css:41]: image
As per the design of homepage they shown only 3 car-cards block on home page And at last they shown 'Go to my wishlist' link for detailed wish list. Could you please restrict it to three car-cards for home page. Again let's confirm this requirement with Kanchan...

Comment (vtMMS) [reply blocks/wishlist/wishlist.css:41]: Hi @mrugesh-adobe, as discussed in teams, this is already restricted when adding to the local storage. I've added yet another restriction in relevant decorate function. Lmk if you got any other concerns.

Comment (mrugesh-adobe) [reply blocks/wishlist/wishlist.css:41]: thanks @vtMMS

**Review by mrugesh-adobe — APPROVED** [review]

**Review by mrugesh-adobe — APPROVED** [review]

**Pull request merged**